src/types/UiaTextRangeBase.cpp

```diff
@@ -2,52 +2,52 @@
 // Licensed under the MIT license.
 
 #include "precomp.h"
-#include "UiaTextRange.hpp"
-#include "ScreenInfoUiaProvider.h"
+#include "UiaTextRangeBase.hpp"
+#include "ScreenInfoUiaProviderBase.h"
 
 using namespace Microsoft::Console::Types;
-using namespace Microsoft::Console::Types::UiaTextRangeTracing;
+using namespace Microsoft::Console::Types::UiaTextRangeBaseTracing;
 
 // toggle these for additional logging in a debug build
-//#define UIATEXTRANGE_DEBUG_MSGS 1
-#undef UIATEXTRANGE_DEBUG_MSGS
-
-IdType UiaTextRange::id = 1;
-
-UiaTextRange::MoveState::MoveState(Microsoft::Console::Render::IRenderData* pData,
-                                   const UiaTextRange& range,
-                                   const MovementDirection direction) :
-    StartScreenInfoRow{ UiaTextRange::_endpointToScreenInfoRow(pData, range.GetStart()) },
-    StartColumn{ UiaTextRange::_endpointToColumn(pData, range.GetStart()) },
-    EndScreenInfoRow{ UiaTextRange::_endpointToScreenInfoRow(pData, range.GetEnd()) },
-    EndColumn{ UiaTextRange::_endpointToColumn(pData, range.GetEnd()) },
+//#define UiaTextRangeBase_DEBUG_MSGS 1

```

Comment:
This looks like an overagressive find/replace hit here and below. (Should be #UIATEXTRANGE_DEBUG_MSGS still)